diff --git a/gralloc/gr_buf_mgr.cpp b/gralloc/gr_buf_mgr.cpp
index c7cfd1df7..456192bc5 100644
--- a/gralloc/gr_buf_mgr.cpp
+++ b/gralloc/gr_buf_mgr.cpp
@@ -253,7 +253,7 @@ gralloc1_error_t BufferManager::ImportHandleLocked(private_handle_t *hnd) {
   }
   // Initialize members that aren't transported
   hnd->size = static_cast<unsigned int>(lseek(hnd->fd, 0, SEEK_END));
-  hnd->offset = 0;
+  //hnd->offset = 0;
   hnd->offset_metadata = 0;
   hnd->base = 0;
   hnd->base_metadata = 0;
diff --git a/gralloc/gr_priv_handle.h b/gralloc/gr_priv_handle.h
index 2acb16cc4..6a4917e47 100644
--- a/gralloc/gr_priv_handle.h
+++ b/gralloc/gr_priv_handle.h
@@ -36,8 +36,6 @@ typedef gralloc1_error_t (*GRALLOC1_PFN_PERFORM)(gralloc1_device_t *device, int
 
 #define PRIV_HANDLE_CONST(exp) static_cast<const private_handle_t *>(exp)
 
-#pragma pack(push, 4)
-
 #ifdef __cplusplus
 struct private_handle_t : public native_handle_t {
 #else
@@ -200,6 +198,5 @@ struct private_handle_t {
   uint64_t GetBackingstore() const { return id; }
 #endif
 };
-#pragma pack(pop)
 
 #endif  // __GR_PRIV_HANDLE_H__
